test/test_hpe_plugin_v2.py

```diff
@@ -83,6 +92,21 @@ def test_clone_compressed_volume(self):
         test = clonevolume_tester.TestCloneCompressedVolume()
         test.run_test(self)
 
+    """
+    CREATE SNAPSHOT REVERT related tests
```

Comment:
Did you mean "REVERT SNAPSHOT related tests"?

Reply:
yes i will change